src/main/java/frc/robot/RobotContainer.java

```diff
@@ -237,7 +242,14 @@ private void configureBindings() {
     OI.getButton(OI.Operator.retractClimber).toggleOnTrue(climberCommandFactory.climb());
 
     new Trigger(() -> OI.Operator.controller.getPOV() == 0).whileTrue(intakeCommand());
+    new Trigger(() -> OI.Operator.controller.getPOV() == 270)
+        .onTrue(new InstantCommand(() -> robotStateManager.setLongRange()));
```

Comment:
Is this the most recent code?  I seem to remember that longrange was unmapped from the commands. @KianChilds